docs/application/native/guides/connectivity/stc.md

```diff
@@ -0,0 +1,227 @@
+# STC
+## Dependencies
+- Tizen 4.0 and higher
+
+STC means Smart Traffic Control. It provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving. STC Library provides APIs fulfilling below mentioned features for application development.
+
+This feature is supported in mobile, tv and werable profile.
```

Comment:
**Spellcheck needed.** 

**Correct Spelling:** Wearable

Reply:
sure will raise another pull request correcting the spelling mistakes.